AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -0,0 +1,19 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+#UI headers
+SCRIPT_CANVAS_UI = "Script Canvas"
+NODE_PALETTE_UI = "Node Palette"
+
+#wait times
+WAIT_TIME_3 = 3
+
+#QtWidgets
+NODE_PALETTE_QT = "NodePalette"
```

Comment:
What's the difference between `NODE_PALETTE_UI` and `NODE_PALETTE_QT`? 

Since there is some reuse in root names, a comment around these two entries will help a reader understand what exactly they intend to reference.